README.md

```diff
@@ -6,17 +6,28 @@
          alt="">
 </a>
 
-Uploadcare Widget is an HTML5 file uploader, a part of the
-[Uploadcare][uc-home]
-ecosystem that fully covers your file handling.
-
-Uploads affect your web or mobile app performance. The widget ensures all of your
-uploads are fast and hit their target.
+Uploadcare Widget is an HTML5 file uploader, a part of the [Uploadcare][uc-home]
+file handling platform.
 
 [![NPM version][badge-npm-img]][badge-npm-url]
 [![Build Status][badge-travis-img]][badge-travis-url]
 [![Uploadcare stack on StackShare][badge-stack-img]][badge-stack-url]
 
+Uploads affect your web or mobile app performance. The widget ensures all of your
+uploads are fast and hit their target.
+
+The widget features:
+
+* Drag&Drop selection.
+* Multiple file upload with individual progress bars.
+* File moderation through validation by file size, extension or MIME type.
+* Image preview and image crop.
+* [Libraries and integrations][uc-libs] for jQuery, JavaScript, Ajax, PHP,
+  Python, Ruby, Rails, Angular, Meteor, and more.
```

Comment:
[Ajax](https://en.wikipedia.org/wiki/Ajax_(programming)) is not integration, remove it. Also, remove Meteor, jQuery

Rails -> Ruby on Rails